healthvault-ios-sdk/Classes/Controllers/MHVBrowserController.m

```diff
@@ -200,4 +159,47 @@ - (void)hideActivitySpinner
     }
 }
 
+#pragma mark - KVO on webView.loading to show spinner
+
+- (void)startObserving
+{
+    [self.webView addObserver:self forKeyPath:kLoadingKeyPath options:kNilOptions context:nil];
+}
+
+- (void)stopObserving
+{
+    [self.webView removeObserver:self forKeyPath:kLoadingKeyPath];
```

Comment:
self.webview nil should do nothing; though the observer:self being nil would cause problems